<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2025, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorMidiControlPluginWidget</class>
 <widget class="QWidget" name="MidiControlPluginWidget">
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ControlTypeTextLabel">
     <property name="text">
      <string>&amp;Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>ControlTypeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ControlTypeComboBox">
     <property name="toolTip">
      <string>MIDI event type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>8</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="ControlChannelTextLabel">
     <property name="text">
      <string>Cha&amp;nnel:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>ControlChannelSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="ControlChannelSpinBox">
     <property name="toolTip">
      <string>MIDI channel</string>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="ControlParamTextLabel">
     <property name="text">
      <string>&amp;Parameter:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>ControlParamComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QComboBox" name="ControlParamComboBox">
     <property name="toolTip">
      <string>MIDI parameter</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QCheckBox" name="ControlLogarithmicCheckBox">
     <property name="text">
      <string>&amp;Logarithmic</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QCheckBox" name="ControlInvertCheckBox">
     <property name="text">
      <string>In&amp;vert</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QCheckBox" name="ControlBipolarCheckBox">
     <property name="text">
      <string>&amp;Bipolar</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="5">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets/>
 <tabstops>
  <tabstop>ControlTypeComboBox</tabstop>
  <tabstop>ControlChannelSpinBox</tabstop>
  <tabstop>ControlParamComboBox</tabstop>
  <tabstop>ControlLogarithmicCheckBox</tabstop>
  <tabstop>ControlInvertCheckBox</tabstop>
  <tabstop>ControlBipolarCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc"/>
 </resources>
 <connections/>
</ui>
